autoid_service/autoid_test.go

```diff
@@ -156,15 +155,8 @@ func TestGRPC(t *testing.T) {
 	etcdCli := cluster.RandClient()
 
 	var addr string
-	var listener net.Listener
-	for port := 10080; ; port++ {
-		var err error
-		addr = fmt.Sprintf("127.0.0.1:%d", port)
-		listener, err = net.Listen("tcp", addr)
-		if err == nil {
-			break
-		}
-	}
+	listener, err := net.Listen("tcp", addr)
+	require.NoError(t, err)
 	defer listener.Close()
```

Comment:
Maybe we need to `addr = listener.Addr().String()`?

Reply:
done